go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	cloud.google.com/go v0.26.0
 	contrib.go.opencensus.io/exporter/stackdriver v0.6.0
 	github.com/Azure/azure-storage-blob-go v0.7.0
+	github.com/Azure/go-autorest/autorest/adal v0.6.0 // indirect
```

Comment:
I don't think you need to change anything because of this, but I wonder why this had to get added

Reply:
probably due to using Go 1.13, it's added in the PR that fixes Thrift anyway :)